website/docs/src/content/component-reference/storage-components/search-reference.mdx

```diff
@@ -0,0 +1,388 @@
+---
+title: Full-text & vector search reference
```

Comment:
being nitpicky but i think the full  "Full-text & vector search" looks clunky in the toc. I'd propose either `Search reference` or `Vector search reference` (full text search is very much a second class citizen imo) 

If we want to highlight we do both, we should just include it in the intro sentence in both articles.

If we update, need to also update this in the concept doc.